lib/mix/tasks/gleam/compile.ex

```diff
@@ -136,37 +135,28 @@ defmodule Mix.Tasks.Gleam.Compile do
       |> Enum.count
 
     if 0 < files do
-      out = "build/dev/erlang/#{app}/build"
+      lib = Path.join(Mix.Project.build_path(), "lib")
+      out = "build/dev/erlang/#{app}"
```

Comment:
Shouldn't this be `Path.join(lib, app)`?

Reply:
No. I haven't gotten things to work properly unless the Erlang files are present in the deps' directories (e.g. tests break, `iex` doesn't load the modules, project environment can only be `dev`).

For now, the `gleam.compile` task copies everything from `out` to `Mix.Project.app_path()` on success.

However, this approach might also align nicely with any plans to ship the Erlang files from Hex.